docs-source/content/userguide/managing-domains/accessing-the-domain/admin-console.md

```diff
@@ -1,6 +1,101 @@
+along with the URL of the WebLogic Server Administration Server's administration port to which you want to connect.
+
+**Note**:  An Administration Server administration port typically is the same as its default port unless either an SSL port or an administration port is configured and enabled.
+
+### Setup
+
+To set up access to WebLogic Server domains running in Kubernetes using the Remote Console:
+
+1. Install, configure, and start the Remote Console according to these [instructions](https://github.com/oracle/weblogic-remote-console/blob/master/site/install_config.md).
+
+1. To give the Remote Console access to an Administration Server running in Kubernetes, you can:
+
+   * Use an [Administration Server `NodePort`](#use-an-administration-server-nodeport).
+
+   * Deploy a load balancer with [ingress path routing rules](#configure-ingress-path-routing-rules).
+
+
+#### Use an Administration Server `NodePort`
+
+For the Remote Console to connect to the Kubernetes WebLogic Server Administration Server’s `NodePort`, use the URL:
+
+```
+http://hostname:adminserver-NodePort/
+```
+
+The `adminserver-NodePort` is the port number of the Administration Server outside the Kubernetes cluster.
+For information about the `NodePort` Service on an Administration Server, see the [Domain resource](https://github.com/oracle/weblogic-kubernetes-operator/blob/master/docs/domains/Domain.md) document.
+
+{{% notice warning %}}
+Exposing administrative, RMI, or T3 capable channels using a Kubernetes `NodePort`
+can create an insecure configuration. In general, only HTTP protocols should be made available externally and this exposure
+is usually accomplished by setting up an external load balancer that can access internal (non-`NodePort`) services.
+For more information, see [T3 channels]({{<relref "/security/domain-security/weblogic-channels#weblogic-t3-channels">}}).
+{{% /notice %}}
+
+#### Configure ingress path routing rules
+
+1. Configure an ingress path routing rule. For more information about ingresses, see the [Ingress]({{< relref "/userguide/managing-domains/ingress/_index.md" >}}) documentation.
+
+   For an example, see the following `path-routing` YAML file for a Traefik load balancer:
+
+   ```yaml
+   apiVersion: traefik.containo.us/v1alpha1
+   kind: IngressRoute
+   metadata:
+     annotations:
+       kubernetes.io/ingress.class: traefik
+     name: traefik-pathrouting-1
+     namespace: weblogic-domain
+   spec:
+     routes:
+     - kind: Rule
+       match: PathPrefix(`/`)
+       services:
+       - kind: Service
+         name: domain1-adminserver
+         namespace: weblogic-domain
+         port: 7001
+   ```
+
+1. For the Remote Console to connect to the Kubernetes WebLogic Server Administration Server, use the URL:
+
+   ```
+   http://${HOSTNAME}:${LB_PORT}/
+   ```
+   Where:
+
+     * `${HOSTNAME}` is where you start up the WebLogic domain.
```

Comment:
For me the HOSTNAME is where my Ingress LB is start up and not the WebLogic domain.  @Maggie?

Reply:
I changed this to ${HOSTNAME} is where the ingress load balancer is running, in this case, the External-IP address of the Traefik load balancer.

Reply:
Removed, "in this case, the External-IP address of the Traefik load balancer."